content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -59,7 +59,8 @@ To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/
       * `--feature-gates="IPv6DualStack=true"`
 
 {{< caution >}}
-If you specify an IPv6 address block larger than a /24 via  `--cluster-cidr` on the command line, that assignment will fail.
+If you specify an IPv6 address block larger than a /64 via  `--cluster-cidr` on the command line, that assignment will fail.
+You can enable IPv6DualStack feature gate for kubelet via adding '--feature-gates="IPv6DualStack=true"' in kubelet's config file.
```

Comment:
@GeorgeGuo2018 this extra line seems like information useful to readers, but it duplicates line 55.

Is it needed?